manifests/application/datamgmt.pp

```diff
@@ -38,17 +31,12 @@
     if $package_name == 'lightblue-data-mgmt-saml-auth' {
         include lightblue::authentication::saml
 
+        if $app_uri == undef {
+            fail("Must define $app_uri if using SAML auth with data management.")
```

Comment:
Just realized I need to use single quotes here to literally put the text, "$app_uri".